src/System.Globalization.Calendars/tests/Misc/MiscCalendars.cs

```diff
@@ -39,35 +37,11 @@ public static void HijriTest()
         }
 
         [Fact]
-        public static void GregorianTest()
-        {
-            GregorianCalendar gCal = new GregorianCalendar();
-            DateTime dTest = gCal.ToDateTime(1600, 1, 1, 0, 0, 0, 0);
-            Assert.Equal(dTest, new DateTime(1600, 1, 1));
-        }
-
-        [Fact]
         public static void JapaneseTest()
         {
             JapaneseCalendar jCal = new JapaneseCalendar();
             DateTime dTest = jCal.ToDateTime(1, 1, 8, 0, 0, 0, 0);
             Assert.Equal(dTest, new DateTime(1989, 1, 8));
         }
-
-        [Fact]
-        public static void KoreanTest()
-        {
-            KoreanCalendar jCal = new KoreanCalendar();
-            DateTime dTest = jCal.ToDateTime(3933, 1, 1, 0, 0, 0, 0);
-            Assert.Equal(dTest, new DateTime(1600, 1, 1));
```

Comment:
Yup sorry again for not leaving a comment explainimg - the test for ToDateTime for calendars like Korean or ThaiBuddhist or Taiwan (where the year is offset) calculate the year by adding/subtracting the offset front be Gregorian year

See
https://github.com/dotnet/corefx/blob/master/src/System.Globalization.Calendars/tests/KoreanCalendar/KoreanCalendarToDateTime.cs

E.g. 1600+2333=3933